### DISCUSSION TITLE
Download Message Text

### DISCUSSION THREAD

**Discussion by notarealemail** [DIGITALCRIMINALS/OnlyFans#241] [Ideas]
Hey, I wonder how feasible it would be to not just scrape OF messages for media but to scrape messages for their text. For example, if you have a whole conversation with someone and want a record of it. Or another example, you've been following someone for a long time and want to reference back to something said - OF's scroll is like 5-10 messages at a time. 

Thoughts?

**Comment by DIGITALCRIMINAL** [comment]
I think the script already does this

**Reply by notarealemail** [reply]
After I posted I had a moment of clarity and realized that I should look at the .db's that are created per user. Sure enough, a full log of messages is found in the sqlite file. I've been able to write queries to find what I've been looking for without issue. It's not terribly attractive, but it's 100% the function I was looking for. 

Thanks for responding and sorry I didn't close the loop on my own post sooner! :)
